pom.xml

```diff
@@ -55,7 +55,7 @@
         <netty-codec-http.version>4.1.45.Final</netty-codec-http.version>
         <liquibase.version>3.4.1</liquibase.version>
         <liquibase-slf4j.version>1.2.1</liquibase-slf4j.version>
-        <cloudfoundry.client.version>1.21.1</cloudfoundry.client.version>
+        <cloudfoundry.client.version>1.22.0-SNAPSHOT</cloudfoundry.client.version>
```

Comment:
Perhaps it would be better to first just release a new version of the cloudfoundry CLI so that we don't forget about this snapshot. :)